### PR TITLE
LPS-203827 Fix small prop

### PR DESCRIPTION
**Pull request by veroglez**
Hi guys!

In Echo team we are making improvements for the `TranslationAdminSelector` component and `input localized` according to this story https://liferay.atlassian.net/browse/LPS-200663. In this story several improvements are proposed:

_Selector_: A new horizontal design for the translation selector

_Dropdown_: A new status for translations, indicating the number of fields that have been translated.

Therefore the following has been done:

**For TranslationAdminSelector.tsx**

- The possibility to display the `TranslationAdminSelector` component with a horizontal design. For this reason, a new prop has been added to this component: `displayType`, with two options, `default` and `horizontal`.

- A new `translationProgress` prop has been included. This prop is used to indicate how many fields are translated in the translation status (see second image from above). It has the following structure:
```
TranslationProgress {
	totalItems: number;
	translatedItems: Record<string, number>;
}
```
- Accessibility improvements
        - Context improvements adding `aria-*`
        - When the selector doesn't show the Translation Manager button it must be a `picker`, and otherwise a `dropdown`
        - For the `dropdown`, when an option is selected the focus will return to the trigger

**For input_localized.js**
- Two attributes have been added to the hidden inputs of the localized input: `data-language-id="{languageId}"` and `data-field-name="{fieldName}"` to make it easier to search from outside the component
- It's necessary to know when the translations are updated, so a `Liferay.fire('inputLocalized:updateTranslationStatus')` is added in the localized input.

Thanks in advance! :slightly_smiling_face:

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.tsx [hunk 1]
```diff
@@ -4,59 +4,135 @@
...
 import {Locale, Translations} from './TranslationAdminContent';
+import TranslationAdminItem from './TranslationAdminItem';
 import TranslationAdminModal from './TranslationAdminModal';
 
+const DISPLAY_TYPE = {
+	default: 'default',
+	horizontal: 'horizontal',
+} as const;
+
+type DisplayType = 'default' | 'horizontal';
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.tsx:24]: A convention for global constants is all caps, so this should be `DEFAULT: 'default'` etc.

A type with a union of values is perfectly fine. An alternative solution that's handy sometimes is an enum, like we use [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/types.ts).

**Review by markocikos — APPROVED** [review]
@veroglez Looks awesome! Very nice use of TS in the code! And very detailed and useful description on the ticket! :sparkles:  

Only a minor nit in comment inline, not blocking.

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.tsx (hunk 1 above)
Comment (victorg1991) [reply modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.tsx:24]: I have resent it here with the uppercase values :D https://github.com/liferay-frontend/liferay-portal/pull/3913

**Comment by victorg1991** [comment]
ci:close

**Pull request closed without merging**